docs/data/charts/heatmap/heatmap.md

```diff
@@ -4,12 +4,15 @@ productId: x-charts
 components: Heatmap, HeatmapPlot, HeatmapItem, DefaultHeatmapTooltip
 ---
 
-# Charts - Heatmap [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')🚧
+# Charts - Heatmap [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan') 🚧
 
 <p class="description">Heatmap charts visually represents data with color variations to highlight patterns and trends across two dimensions.</p>
 
 :::warning
-The Heatmap Chart component isn't stable. Don't hesitate to open issues to give feedback.
+The Heatmap Chart component is part of the pro package which is **not yet** released.
+
+You can test demos, and codesandboxes.
```

Comment:
Are you sure? 🤔 
The package is not released. 🤔 
```suggestion
You can test demos.
```

Reply:
My bad, it's because in preview deployment, they use a build from the PR